new-frontiers-14/frontier-station-14
# Fix ammunition icons in vendors

## About the PR



Add icons for each ammunition box type, and for the shotgun ammunition dispenser boxes.

## Why / Balance


Tricky to read ammunition types in the liberation station, especially for shotguns.  No more "why machine show red box but give blue box?" at LessLethalVend/Lib. Station pending #1388.

## How to test


1. Spawn LessLethalVend, Liberation Station, cryptographic sequencer.
2. Use cryptographic sequencer on both vendors.
3. View and buy each ammunition type.  The in-game sprite should match the picture in the vendor (no empty boxes, correct colours).

## Media


With change:
![liberation_station_with_change](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/ce56bd08-04d0-4102-8c1e-45986ccd3d5b)

Without change:
![liberation_station_without_change](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/55e0d70f-0859-431a-b6b4-65b100503947)


## Breaking changes


**Changelog**



- fix: Liberation Station and LessLethalVend machines now show correct ammunition icons.
